test/MezzioInstallerTest/HomePageResponseTest.php

```diff
@@ -56,6 +74,43 @@ class HomePageResponseTest extends OptionalPackagesTestCase
         'laminas-view' => [3, LaminasViewRenderer::class],
     ];
 
+    private $expectedRendererAttributes = [
+        PlatesRenderer::class => [
+            'templateName' => 'Plates',
+            'templateDocs' => 'http://platesphp.com/',
+        ],
+        TwigRenderer::class => [
+            'templateName' => 'Twig',
+            'templateDocs' => 'http://twig.sensiolabs.org/documentation',
+        ],
+        LaminasViewRenderer::class => [
+            'templateName' => 'Laminas View',
+            'templateDocs' => 'https://docs.laminas.dev/laminas-view/',
+        ],
+    ];
+
+    // $routerOption, $routerClass
+    private $routerTypes = [
+        'aura-router'    => [1, AuraRouter::class],
+        'fastroute'      => [2, FastRouteRouter::class],
+        'laminas-router' => [3, LaminasRouter::class],
+    ];
+
+    private $expectedRouterAttributes = [
+        AuraRouter::class => [
+            'routerName' => 'Aura.Router',
+            'routerDocs' => 'http://auraphp.com/packages/2.x/Router.html',
```

Comment:
use v3 actually, see https://github.com/mezzio/mezzio-aurarouter/blob/a22e92f0e79445229e57a612eb5acf2e841d2cb6/composer.json#L37

probably routerDocs link can be changed to http://auraphp.com/packages/3.x/Router/

Reply:
Hello @samsonasik, I have to wait for HomePageHandler-updating PRs to be pulled-in (for aura-di v4 as well), otherwise the response-inspecting tests would fail.  
Pinging @michalbundyra as well: maybe in a future version we could add a support class (or a HomePageHandler public constant) defining name and docs based on the container/router/renderer FQCN, so that we have one "place" to make those changed. This would simplify updating the test cases a lot. kind regards.